<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostTimeEditDialog</class>
 <widget class="QDialog" name="PostTimeEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="formatGroupBox">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="secondRadioButton">
        <property name="text">
         <string>ss sec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="minuteSecondRadioButton">
        <property name="text">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hourMinuteSecondRadioButton">
        <property name="text">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dayHourMinuteSecondRadioButton">
        <property name="text">
         <string>dd:hh:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="vtkTextPropertySettingWidget" name="fontSettingWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vtkTextPropertySettingWidget</class>
   <extends>QWidget</extends>
   <header>guibase/vtktextpropertysettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PostTimeEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PostTimeEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
